model/histogram/float_histogram_test.go

```diff
@@ -2314,6 +2375,9 @@ func TestFloatHistogramAdd(t *testing.T) {
 }
 
 func testHistogramAdd(t *testing.T, a, b, expected *FloatHistogram, expErrMsg string, expCounterResetCollision bool) {
+	require.NoError(t, a.Validate(), "a")
+	require.NoError(t, b.Validate(), "b")
```

Comment:
It is difficult enough to reason about histogram operations in test cases, and having invalid inputs makes it only harder. Because of that, decided to add validation of test inputs. Had to adjust some existing test cases.